utilities/check_mrTrix_Version.m

```diff
@@ -0,0 +1,19 @@
+function mrTrixVersion = check_mrTrix_Version
```

Comment:
ok, I'll change and commit.

What do you think about this? You just launch your matlab with this command:
LD_PRELOAD="/usr/local/eb/software/GCC/4.9.3/lib64/libstdc++.so.6" matlab &
you can even create an alias in bash_profile:
alias matlab_ld='LD_PRELOAD="/usr/local/eb/software/GCC/4.9.3/lib64/libstdc++.so.6"
matlab &'

in any case, I'll edit it and commit it.

On Tue, Jul 12, 2016 at 2:48 AM Jason D. Yeatman <notifications@github.com>
wrote:

> In utilities/check_mrTrix_Version.m
> <https://github.com/yeatmanlab/AFQ/pull/11#discussion_r70362495>:
>
> > @@ -0,0 +1,19 @@
> > +function mrTrixVersion = check_mrTrix_Version
>
> In the previous versions I just used which to see if the functions were
> installed in the system. This will work fine But without resetting the
> library paths none of these functions will run. In your new version you
> check the output of mrview to get the version number. mrview will not give
> the correct output on an ubuntu system without setting the library paths
> properly. One solution would be to check the mrtrix version without calling
> mrview. For example just use system('which ') with the name of an mrtrix3
> specific function. Jason D. Yeatman, PhD Assistant Professor, Institute for
> Learning & Brain Sciences (I-LABS) Department of Speech & Hearing Sciences
> University of Washington http://brainandeducation.com On Mon, Jul 11,
> 2016 at 5:18 PM, Garikoitz Lerma-Usabiaga < notifications@github.com>
> wrote:
> … <#m_3342909447770579329_>
> In utilities/check_mrTrix_Version.m <#11 (comment)
> <https://github.com/yeatmanlab/AFQ/pull/11#discussion_r70359943>>: > @@
> -0,0 +1,19 @@ > +function mrTrixVersion = check_mrTrix_Version hmmmm not in
> my case... Can you please send me the outputs of both system calls I sent
> you before? system('mrconvert -version') system('gcc -v') On Tue, Jul 12,
> 2016 at 2:07 AM Jason D. Yeatman ***@***.***> wrote: …
> <#m_-573941099128601710_> In utilities/check_mrTrix_Version.m <#11
> <https://github.com/yeatmanlab/AFQ/pull/11> (comment) <#11 (comment)
> <https://github.com/yeatmanlab/AFQ/pull/11#discussion_r70359065>>>: > @@
> -0,0 +1,19 @@ > +function mrTrixVersion = check_mrTrix_Version Without this
> call to AFQ_mrtrix_set_ld_path it is impossible to make any system call to
> mrtrix. So this means that any function to check if mrtrix is installed
> fails. Jason D. Yeatman, PhD Assistant Professor, Institute for Learning &
> Brain Sciences (I-LABS) Department of Speech & Hearing Sciences University
> of Washington http://brainandeducation.com On Mon, Jul 11, 2016 at 4:57
> PM, Garikoitz Lerma-Usabiaga < ***@***.***> wrote: …
> <#m_8673309125022752045_> In utilities/check_mrTrix_Version.m <#11
> <https://github.com/yeatmanlab/AFQ/pull/11> <#11
> <https://github.com/yeatmanlab/AFQ/pull/11>> (comment) <#11
> <https://github.com/yeatmanlab/AFQ/pull/11> (comment) <#11 (comment)
> <https://github.com/yeatmanlab/AFQ/pull/11#discussion_r70358150>>>>: > @@
> -0,0 +1,19 @@ > +function mrTrixVersion = check_mrTrix_Version Yes, I wrote
> a note. I don't have a problem with it in OSX, and in two different linux
> boxes (both Red Hat), one normal linux box and the other one part of a
> cluster. Are you using ubuntu? This is the note: % Jason, I don't have the
> AFQ_mrtrix_set_ld_path problem (in linux/OSX), I get rid of it with % the
> matlab call itself. Furthermore, it would be circular, since that function
> % is using the version information as well. We could us a function call
> used in % both versions. Please edit this as it is working for you and I %
> will override it in my version if necessary. I don't understand what your
> problem is, can you send me the error please? … <#m_5681299657681076971_>
> On Tue, Jul 12, 2016 at 12:56 AM dstrodtman ***@***.***> wrote: In
> utilities/check_mrTrix_Version.m <#11
> <https://github.com/yeatmanlab/AFQ/pull/11> <#11
> <https://github.com/yeatmanlab/AFQ/pull/11>> <#11
> <https://github.com/yeatmanlab/AFQ/pull/11> <#11
> <https://github.com/yeatmanlab/AFQ/pull/11>>> (comment) <#11
> <https://github.com/yeatmanlab/AFQ/pull/11> <#11
> <https://github.com/yeatmanlab/AFQ/pull/11>> (comment) <#11
> <https://github.com/yeatmanlab/AFQ/pull/11> (comment) <#11 (comment)
> <https://github.com/yeatmanlab/AFQ/pull/11#discussion_r70352285>>>>>: >
> @@ -0,0 +1,19 @@ > +function mrTrixVersion = check_mrTrix_Version This
> doesn't look like it has been fixed. For this function to run properly we
> need to first call AFQ_mrtrix_set_ld_path. However AFQ_mrtrix_set_ld_path
> takes as an input, the output of the present function. Gari, did this work
> for you on linux? — You are receiving this because you modified the
> open/close state. Reply to this email directly, view it on GitHub <
> https://github.com/yeatmanlab/AFQ/pull/11/files/7fc540e2aad16177aeded65d4e6516e6f78cf07e#r70352285>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/AEWQV5c4XAECeJhde8WXaJsfh1eYoxrYks5qUsoggaJpZM4JGbEU>
> . — You are receiving this because you commented. Reply to this email
> directly, view it on GitHub <
> https://github.com/yeatmanlab/AFQ/pull/11/files/7fc540e2aad16177aeded65d4e6516e6f78cf07e#r70358150>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ABsYyh8UDRjPcpSV02PrCA23b81jpIUWks5qUthSgaJpZM4JGbEU>
> . — You are receiving this because you modified the open/close state. Reply
> to this email directly, view it on GitHub <
> https://github.com/yeatmanlab/AFQ/pull/11/files/7fc540e2aad16177aeded65d4e6516e6f78cf07e#r70359065>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/AEWQV4M9A7NC_D10jp3vaSKStrAakiByks5qUtrcgaJpZM4JGbEU>
> . — You are receiving this because you commented. Reply to this email
> directly, view it on GitHub <
> https://github.com/yeatmanlab/AFQ/pull/11/files/7fc540e2aad16177aeded65d4e6516e6f78cf07e#r70359943>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ABsYyuFUfpYiJZEKkmZNckS1LKtpFkptks5qUt1ggaJpZM4JGbEU>
> .
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/yeatmanlab/AFQ/pull/11/files/7fc540e2aad16177aeded65d4e6516e6f78cf07e#r70362495>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEWQVz5fY5xOHlMjeEuEIVQetI74n-8Wks5qUuRJgaJpZM4JGbEU>
> .
>